diff --git a/SpriteBot.py b/SpriteBot.py
index 9fdc770..2454276 100644
--- a/SpriteBot.py
+++ b/SpriteBot.py
@@ -2201,6 +2201,7 @@ class SpriteBot:
         prefix = self.config.servers[str(msg.guild.id)].prefix
         if len(args) == 0:
             return_msg = "**Commands**\n" \
+                  f"`{prefix}add` - Adds a Pokemon or forme to the current list\n" \
                   f"`{prefix}sprite` - Get the Pokemon's sprite sheet\n" \
                   f"`{prefix}portrait` - Get the Pokemon's portrait sheet\n" \
                   f"`{prefix}recolorsprite` - Get the Pokemon's sprite sheet in a form for easy recoloring\n" \
@@ -2217,7 +2218,17 @@ class SpriteBot:
 
         else:
             base_arg = args[0]
-            if base_arg == "listsprite":
+            if base_arg == "add":
+                return_msg = "**Command Help**\n" \
+                             f"`{prefix}add <Pokemon Name> [Form Name]`\n" \
+                             "Adds a Pokemon to the dex, or a form to the existing Pokemon.\n" \
+                             "`Pokemon Name` - Name of the Pokemon\n" \
+                             "`Form Name` - [Optional] Form name of the Pokemon\n" \
+                             "**Examples**\n" \
+                             f"`{prefix}add Calyrex`\n" \
+                             f"`{prefix}add Mr_Mime Galar`\n" \
+                             f"`{prefix}add Missingno_ Kotora`"
+            elif base_arg == "listsprite":
                 return_msg = "**Command Help**\n" \
                              f"`{prefix}listsprite <Pokemon Name>`\n" \
                              "List all sprites related to a Pokemon.  This includes all forms, gender, and shiny variants.\n" \
@@ -2391,7 +2402,6 @@ class SpriteBot:
         prefix = self.config.servers[str(msg.guild.id)].prefix
         if len(args) == 0:
             return_msg = "**Approver Commands**\n" \
-                  f"`{prefix}add` - Adds a Pokemon or forme to the current list\n" \
                   f"`{prefix}delete` - Deletes an empty Pokemon or forme\n" \
                   f"`{prefix}rename` - Renames a Pokemon or forme\n" \
                   f"`{prefix}addgender` - Adds the female sprite/portrait to the Pokemon\n" \
@@ -2418,17 +2428,7 @@ class SpriteBot:
 
         else:
             base_arg = args[0]
-            if base_arg == "add":
-                return_msg = "**Command Help**\n" \
-                             f"`{prefix}add <Pokemon Name> [Form Name]`\n" \
-                             "Adds a Pokemon to the dex, or a form to the existing Pokemon.\n" \
-                             "`Pokemon Name` - Name of the Pokemon\n" \
-                             "`Form Name` - [Optional] Form name of the Pokemon\n" \
-                             "**Examples**\n" \
-                             f"`{prefix}add Calyrex`\n" \
-                             f"`{prefix}add Mr_Mime Galar`\n" \
-                             f"`{prefix}add Missingno_ Kotora`"
-            elif base_arg == "delete":
+            if base_arg == "delete":
                 return_msg = "**Command Help**\n" \
                              f"`{prefix}delete <Pokemon Name> [Form Name]`\n" \
                              "Deletes a Pokemon or form of an existing Pokemon.  " \
@@ -2826,7 +2826,7 @@ async def on_message(msg: discord.Message):
             elif base_arg == "autocolor":
                 await sprite_bot.tryAutoRecolor(msg, args[1:], "portrait")
                 # authorized commands
-            elif base_arg == "add" and authorized:
+            elif base_arg == "add":
                 await sprite_bot.addSpeciesForm(msg, args[1:])
             elif base_arg == "delete" and authorized:
                 await sprite_bot.removeSpeciesForm(msg, args[1:])
